.github/workflows/checks.yml

```
@@ -169,7 +169,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        go-version: ["1.22.12", "1.23.6"] # 1.20 is interpreted as 1.2 without quotes
+        go-version: ["1.23.6", "1.24.0"] # 1.20 is interpreted as 1.2 without quotes
```

Use `x` for the patch version to always use the latest one.
```suggestion
        go-version: ["1.23.x", "1.24.x"] # 1.20 is interpreted as 1.2 without quotes
```

---

Good call, thanks!